### DISCUSSION TITLE
abacustest functionality issue

### DISCUSSION THREAD

**Discussion by meishengme** [deepmodeling/abacus-develop#5341] [General]
### Describe the Testing Issue

When using abacustest to convert an abacus input file to a qe input file, there are some parameters that are not converted：
1、dft_functional→input_dft;
2、nx  →  nr1
      ny  →  nr2
      nz  →  nr3
3、abacustest does not recognize the parameter： latname 
      can not convert to qe input file 
4、abacustest does not recognize the parameter：smearing_method=fd
         It is missing when converting to a qe input file：
         occupations = smearing
          smearing = fd
          degauss = 0.002
5、fixed_axes  →  cell_dofree
      stress_thr  →  press_conv_thr
6、When performing structural optimization, when converting to a qe input file, the structure information of whether the point is fixed or not is fixed to a certain atom will be ignored
7、abacustest does not recognize MD calculation settings
8、abacustest does not recognize Monkhorst-Pack

### Task list for Issue attackers (only for developers)

- [ ] Understand the testing issue described by the developer.
- [ ] Review the specific test case, expected and actual results, and any error messages.
- [ ] Identify the root cause of the test failure or issue.
- [ ] If a possible solution is suggested, evaluate its feasibility and effectiveness.
- [ ] Implement a fix for the test failure or issue, or create a new test case if needed.
- [ ] Verify that the fix resolves the testing issue and the test case passes.
- [ ] Review and update any relevant documentation, such as test plans or user guides.
- [ ] Ensure the testing issue is resolved and close the ticket.
- [ ] Share any lessons learned or best practices with the team to prevent similar issues in the future.

**Comment by pxlxingliang** [comment]
The convention of 2/4/5 is support now. 
For 3, defining cell by parameter "latname" is not a common way, and the conversion will be quite complex, requiring the STRU files. We will not consider supporting this feature at the moment.
For 1/6/7/8, I will support them in the future.
